Firmware/monorail/keymaps/vial/vial.json

```diff
@@ -1 +1,345 @@
-{ "name": "Monorail", "vendorId": "4B44", "productId": "4D72", "lighting": "qmk_rgblight", "matrix": { "rows": 5, "cols": 12 }, "layouts": { "labels":[ [ "Bottom Row", "Diesel Southpaw + Arrows", "Diesel Arrows", "Diesel Southpaw", "Diesel OG", "Jet Split Space", "Jet Big Bar", "Steam 300 + 300", "Steam 275 + 300", "Steam Big Bar" ], "Remove Top Row" ], "keymap": [ [ { "x": 0.5, "d": true }, "4,0\n\n\n1,1", { "d": true }, "4,1\n\n\n1,1", { "d": true }, "4,2\n\n\n1,1", { "d": true }, "4,3\n\n\n1,1", { "d": true }, "4,4\n\n\n1,1", { "d": true }, "4,5\n\n\n1,1" ], [ { "y": 0.25, "x": 0.5 }, "4,0\n\n\n1,0", "4,1\n\n\n1,0", "4,2\n\n\n1,0", "4,3\n\n\n1,0", "4,4\n\n\n1,0", "4,5\n\n\n1,0" ], [ "0,0", "0,1", "0,2", "0,3", "0,4", "0,5", "0,6", "0,7", "0,8", "0,9", "0,10", { "w": 1.75 }, "0,11" ], [ { "w": 1.25 }, "1,0", "1,1", "1,2", "1,3", "1,4", "1,5", "1,6", "1,7", "1,8", "1,9", "1,10", { "w": 1.5 }, "1,11" ], [ { "w": 1.75 }, "2,0", "2,1", "2,2", "2,3", "2,4", "2,5", "2,6", "2,7", "2,8", "2,9", "2,10", "2,11" ], [ "3,0\n\n\n0,0", "3,1\n\n\n0,0", "3,2\n\n\n0,0", "3,3\n\n\n0,0", { "w": 2.25 }, "3,4\n\n\n0,0", { "w": 2 }, "3,6\n\n\n0,0", { "w": 1.5 }, "3,8\n\n\n0,0", "3,9\n\n\n0,0", "3,10\n\n\n0,0", "3,11\n\n\n0,0" ], [ { "y": 0.25, "w": 1.25 }, "3,0\n\n\n0,1", { "w": 1.5 }, "3,1\n\n\n0,1", { "w": 1.25 }, "3,3\n\n\n0,1", { "w": 2.25 }, "3,4\n\n\n0,1", { "w": 2 }, "3,6\n\n\n0,1", { "w": 1.5 }, "3,8\n\n\n0,1", "3,9\n\n\n0,1", "3,10\n\n\n0,1", "3,11\n\n\n0,1" ], [ "3,0\n\n\n0,2", "3,1\n\n\n0,2", "3,2\n\n\n0,2", "3,3\n\n\n0,2", { "w": 2.25 }, "3,4\n\n\n0,2", { "w": 2 }, "3,6\n\n\n0,2", { "w": 1.25 }, "3,8\n\n\n0,2", { "w": 1.5 }, "3,9\n\n\n0,2", { "w": 1.75 }, "3,11\n\n\n0,2" ], [ { "w": 1.25 }, "3,0\n\n\n0,3", { "w": 1.5 }, "3,1\n\n\n0,3", { "w": 1.25 }, "3,3\n\n\n0,3", { "w": 2.25 }, "3,4\n\n\n0,3", { "w": 2 }, "3,6\n\n\n0,3", { "w": 1.25 }, "3,8\n\n\n0,3", { "w": 1.5 }, "3,9\n\n\n0,3", { "w": 1.75 }, "3,11\n\n\n0,3" ], [ { "w": 1.25 }, "3,0\n\n\n0,4", "3,1\n\n\n0,4", { "w": 1.25 }, "3,2\n\n\n0,4", { "w": 2.75 }, "3,4\n\n\n0,4", { "w": 2.25 }, "3,6\n\n\n0,4", { "w": 1.25 }, "3,8\n\n\n0,4", "3,9\n\n\n0,4", "3,10\n\n\n0,4", "3,11\n\n\n0,4" ], [ { "w": 1.25 }, "3,0\n\n\n0,5", "3,1\n\n\n0,5", { "w": 1.25 }, "3,2\n\n\n0,5", { "w": 6.25 }, "3,6\n\n\n0,5", "3,9\n\n\n0,5", "3,10\n\n\n0,5", "3,11\n\n\n0,5" ], [ "3,0\n\n\n0,6", { "w": 1.25 }, "3,1\n\n\n0,6", "3,2\n\n\n0,6", { "w": 3 }, "3,4\n\n\n0,6", { "w": 3 }, "3,6\n\n\n0,6", { "w": 1.25 }, "3,9\n\n\n0,6", "3,10\n\n\n0,6", { "w": 1.25 }, "3,11\n\n\n0,6" ], [ { "w": 1.25 }, "3,0\n\n\n0,7", "3,1\n\n\n0,7", { "w": 1.25 }, "3,2\n\n\n0,7", { "w": 2.75 }, "3,4\n\n\n0,7", { "w": 3 }, "3,6\n\n\n0,7", { "w": 1.25 }, "3,9\n\n\n0,7", "3,10\n\n\n0,7", { "w": 1.25 }, "3,11\n\n\n0,7" ], [ { "w": 1.25 }, "3,0\n\n\n0,8", "3,1\n\n\n0,8", "3,2\n\n\n0,8", { "w": 6.25 }, "3,6\n\n\n0,8", "3,9\n\n\n0,8", "3,10\n\n\n0,8", { "w": 1.25 }, "3,11\n\n\n0,8" ] ] } }
\ No newline at end of file
+{
```

Comment:
Removing the whitespace from this file will use less space in the compiled firmware.